health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CycleConfiguration.js

```
@@ -156,11 +157,12 @@
   // const saved =
   //   Digit.SessionStorage.get("HCM_CAMPAIGN_MANAGER_FORM_DATA")?.HCM_CAMPAIGN_CYCLE_CONFIGURE?.cycleConfigure ||
   //   campaignData?.additionalDetails?.cycleData || filteredDeliveryConfig?.cycleConfig;
-  const sessionData = Digit.SessionStorage.get("HCM_CAMPAIGN_MANAGER_FORM_DATA")?.HCM_CAMPAIGN_CYCLE_CONFIGURE?.cycleConfigure;
+  // const sessionData = Digit.SessionStorage.get("HCM_CAMPAIGN_MANAGER_FORM_DATA")?.HCM_CAMPAIGN_CYCLE_CONFIGURE?.cycleConfigure;
+  const sessionData = Digit.SessionStorage.get("HCM_ADMIN_CONSOLE_DATA")?.additionalDetails?.cycleData;
   const campaignCycleData = campaignData?.additionalDetails?.cycleData;
   const filteredCycleConfig = filteredDeliveryConfig?.cycleConfig;
 
-  const saved = sessionData?.cycleData?.length > 0 ? sessionData : campaignCycleData?.cycleData?.length > 0 ? campaignCycleData : filteredCycleConfig;
+  let saved = sessionData?.cycleData?.length > 0 ? sessionData :  filteredCycleConfig;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'saved' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)